### PR TITLE
add getModel method

### PR DESCRIPTION
**Pull request by mbarekh**

### CODE REVIEW

File: src/communication/xcConnection.ts [hunk 1]
```diff
@@ -3,6 +3,7 @@ import { ApiConfiguration } from "configuration/apiConfiguration";
 import { DefaultApiConfigurationParser } from "configuration/apiConfigurationParser";
 
 export interface Connection {
+    getModel(xcApiName: string, serverUrl: string, getModelListener: (model: any) => void);
```
Comment (maurelio1234) [inline src/communication/xcConnection.ts]: 1. is it possible to be more precise on "any"?
2. getModelListener... looking at the other methods... shouldn't it be modelListener ?

File: src/communication/xcConnection.ts [hunk 2]
```diff
@@ -18,15 +19,29 @@ export class DefaultConnection implements Connection {
...
 
+    getModel(xcApiName: string, serverUrl: string, getModelListener: (model: any) => void) {
+        let session = SessionFactory(serverUrl, null, null);
+        let openListener = (_: Event) => {
+            session.privateSubscriber.getModel(xcApiName, (model: any) => {
+                getModelListener(model);
+                session.close();
+            });
+        };
+        let errorListener = (_: Error) => {
```
Comment (maurelio1234) [inline src/communication/xcConnection.ts]: why ignoring the error object?

Comment (maurelio1234) [inline src/communication/xcConnection.ts]: add it to the console message

Comment (mbarekh) [inline src/communication/xcConnection.ts]: done

File: src/communication/xcConnection.ts (hunk 1 above)
Comment (mbarekh) [inline src/communication/xcConnection.ts]: We can be more precise on type "any". It is the purpose of the issue #42

**Pull request merged**